crates/bevy_pbr/src/prepass/prepass.wgsl

```diff
@@ -32,14 +32,29 @@ fn morph_vertex(vertex_in: Vertex) -> Vertex {
     }
     return vertex;
 }
+
+
+#ifdef MOTION_VECTOR_PREPASS
+fn morph_previous_position(vertex: Vertex) -> vec3<f32> {
+    var vertex_position = vertex.position;
+    let weight_count = bevy_pbr::morph::layer_count();
+    for (var i: u32 = 0u; i < weight_count; i++) {
+        let weight = bevy_pbr::morph::previous_weight_at(i);
+        if weight != 0.0 {
```

Comment:
This isn't new, so not blocking, but I wonder if this branch is helpful or harmful. It probably has to run both branches regardless I would think?

Reply:
That's a really good question. My GPU-fu is not strong enough to be able to answer it. I don't remember the motivation behind it.